main.cpp

```diff
@@ -491,6 +668,33 @@ int AmplitudeEnvelope::changeDryWetMix(int wetValue)
     return dryValue - wetValue;
 }
 
+void AmplitudeEnvelope::emergencyVolumeLimiter()
+{
+    std::vector<double> buffer;
+
+    static std::default_random_engine engine;
+    static std::uniform_int_distribution<unsigned> distribution(0, 2);
+
+
+    for(size_t i = 0; i < 20; ++i)
+    {
+        buffer.push_back(distribution(engine));
+    }
+
+    std::cout << "Buffer sample value: ";
+    for (auto c : buffer)
```

Comment:
You haven't learned about references yet in this course.
I bet you are assuming that `c` is an element in `buffer`.
it is not.   
Don't use stuff you haven't learned about in the course.
switch to a regular `for()` loop and index the elements directly.
- [ ] revise

Reply:
I thought c was  the cotents of buffer.begin() to buffer.end()